guides/v2.2/config-guide/varnish/config-varnish-advanced.md

```diff
@@ -94,12 +94,13 @@ After you've recompiled, you can install the Saint mode {% glossarytooltip c1e42
 
 <li><p>Build the source code with autotools:</p>
 
-<p><code>sudo apt-get install libvarnishapi-dev || sudo yum install varnish-libs-devel</code></p>
-<p><code>./bootstrap   # If running from git.</code></p>
-<p><code>./configure</code></p>
-<p><code>make</code></p>
-<p><code>make check   # optional</code></p>
-<p><code>sudo make install</code></p>
+    sudo apt-get install libvarnishapi-dev || sudo yum install varnish-libs-devel
```

Comment:
This actually doesn't work, as the code is within an HTML list. To remedy this, convert the list to markdown, and then add backticks to your code. Like this:

```
1. Obtain the source code from [Varnish modules](https://github.com/varnish/varnish-modules) . Clone the git version (master version) since the 0.9.x versions contain a source code error.

1. Build the source code with autotools:

    ``` bash
    sudo apt-get install libvarnishapi-dev || sudo yum install varnish-libs-devel
    ./bootstrap   # If running from git.
    ./configure
    make
    make check   # optional
    sudo make install
    ```
```